pints/toy/_twisted_gaussian_banana.py

```diff
@@ -66,6 +66,7 @@ def __call__(self, x):
         y = np.array(x, copy=True, dtype='float')
         y[0] = float(y[0]) / np.sqrt(self._V)
         y[1] += self._b * ((x[0] ** 2) - self._V)
+        y = y.reshape(self._n_parameters)
```

Comment:
Would make more sense to do it with `x` here (so on line 66), so that the error comes from somewhere with a variable name that the user recognises.